packages/blockprotocol/core.d.ts

```diff
@@ -22,22 +22,79 @@ export type BlockMetadataRepository =
   | string;
 
 export type BlockMetadata = {
-  author?: string | null;
-  default?: JSONObject | null;
-  description?: string | null;
-  displayName?: string | null;
-  examples?: JSONObject[] | null;
-  externals?: Record<string, string> | null;
-  icon?: string | null;
-  image?: string | null;
-  license?: string | null;
-  name?: string | null;
-  protocol?: string | null;
```

Comment:
Please maintain the nullability of all fields except `null`, `source`, `version` and `protocol` - the rest are not required by the spec.